src/containers/JournalPageContainer/index.jsx

```diff
@@ -12,6 +12,7 @@ const mapStateToProps = state => (
     fetchPageSuccess: state.page.error,
     nextPage: state.page.page.next_page_id,
     previousPage: state.page.page.previous_page_id,
+    is_preview: state.is_preview,
```

Comment:
We don't need this because the default proptype is false.

Reply:
fixed